### PR TITLE
Created command-line UI

### PR DESCRIPTION
**Pull request by mullenlucas**
## Implemented features:

- Create a `app.rb` file that will serve as your console app entry-point. It should have methods that do the following:
  - List all books. (1)
  - List all people. (2)
  - Create a person (teacher or student). (3)
  - Create a book. (4)
  - Create a rental. (5)
  - List all rentals for a given person id. (6)
- In your `main.rb` define the entry point, this will be a method called `main` that is invoked at the end of your file. This method does the following:
  - Present the user with a list of options to perform. [1-7]
  - Lets users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app. (7)

### CODE REVIEW

File: entities/person.rb [hunk 1]
```diff
@@ -35,16 +35,3 @@ def of_age?
     @age >= 18
   end
 end
-
-# testing
```
Comment (HAMMAS-SALEEM) [inline entities/person.rb:39]: image

- [x] You have added a LICENSE file in your project :blush: and i can see a link in readme file for the licnese file :+1: but the link is broken :fearful: Kindly fix this link to make your readme descriptive :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @mullenlucas :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following GitFlow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: entities/person.rb (hunk 1 above)
Comment (mullenlucas) [reply entities/person.rb:39]: Thank you for pointing this out ! It's the first time I create a project with the license template and forgot to update the readme link value

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- 

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by GzimAsani — APPROVED** [review]

**Pull request merged**